static/app/views/explore/toolbar/toolbarVisualize.tsx

```diff
@@ -79,7 +79,7 @@ export function ToolbarVisualize({}: ToolbarVisualizeProps) {
         </ToolbarHeaderButton>
       </ToolbarHeader>
       {parsedVisualizes.map((parsedVisualize, index) => (
-        <VisualizeOption key={index}>
+        <ToolbarRow rows={visualizes} setRows={setVisualizes} index={index} key={index}>
```

Comment:
Can you undo the changes to this file here? We're discussing some changes that would be incompatible with this helper component.

Reply:
Okay so will just keep the delete to groupBy for now